py/dml/compat.py

```diff
@@ -118,6 +118,23 @@ class shared_logs_on_device(CompatFeature):
     last_api_version = api_6
 
 
+@feature
+class suppress_WLOGMIXUP(CompatFeature):
+    '''This compatibility feature makes it so the warning `WLOGMIXUP` is
+    suppressed by default. `WLOGMIXUP` warns about usages of a common faulty
+    pattern which results in broken log statements &mdash; for more
+    information, see the documentation of `WLOGMIXUP` in the
+    [Messages](messages.html) section of the DML 1.4 Reference Manual.
```

Comment:
"of the ... manual" can be omitted, since this text is within an appendix in the same manual.

Reply:
Sure.